src/app/App.tsx

```diff
@@ -2,18 +2,51 @@ import * as React from "react";
 import {BrowserRouter as Router, Route, Switch} from "react-router-dom";
 import {ChampionPage} from "./components/champion";
 import {Header} from "./components/commons/Header";
+import {createMuiTheme, ThemeProvider} from "@material-ui/core/styles";
+import {CssBaseline, Paper} from "@material-ui/core";
+
+const theme = createMuiTheme({
+    typography: {
+        fontFamily: [
+            "Rubik",
+            "sans-serif",
+            "Roboto",
+        ].join(',')
+    },
+    palette: {
+        type: 'dark',
+        common:  {
+            black: '#434449',
+            white: '#EAEBF0'
+        },
+        primary: {
+            main: '#3a22a5',
+        },
+        secondary: {
+            main: '#e1bee7',
+        },
+        background: {
+            default: '#1e1159',
+        },
+    },
+});
 
 export const App = () => {
     return (
-        <Router>
-            <Header/>
-            <Switch>
-                <Route path="/champion">
-                    <ChampionPage/>
-                </Route>
-                <Route path="/">
-                </Route>
-            </Switch>
-        </Router>
+        <ThemeProvider theme={theme}>
+            <CssBaseline />
```

Comment:
I think you are missing a GlobalStyleProvider. It is why you have to use the colors of your theme in some component

Reply:
GlobalStyleProvider has been added accordingly